### ISSUE TITLE
Crash in requestPermissions()

### ISSUE THREAD

**Issue by adrynov** [sagar-viradiya/eazypermissions#28]
Used your code in a new Android application:

```
 requestPermissions(Manifest.permission.ACCESS_FINE_LOCATION) {
            requestCode = 1
            resultCallback = {
                handlePermissionsResult(this)
            }
        }


```

It crashes with:
java.lang.NoSuchMethodError: No virtual method getRequestCode()I in class Lcom/eazypermissions/common/model/PermissionResult; or its super classes (declaration of 'com.eazypermissions.common.model.PermissionResult' appears in base.apk)
        at com.eazypermissions.dsl.PermissionManager.onPermissionResult(PermissionManager.kt:35)
        at com.eazypermissions.common.BasePermissionManager.requestPermissions(BasePermissionManager.kt:82)
        at com.eazypermissions.dsl.PermissionManager.access$requestPermissions(PermissionManager.kt:30)
        at com.eazypermissions.dsl.PermissionManager$Companion._requestPermissions(PermissionManager.kt:129)

The "requestCode" is shown as unresolved in 

```
override fun onPermissionResult(permissionResult: PermissionResult) {
        callbackMap[permissionResult.requestCode]?.let {
            permissionResult.it()
        }
        callbackMap.remove(permissionResult.requestCode)
    }
```

**Comment by AlbertoChamorro** [comment]
Hi, I think the error is in the Permission class Result of version 2.0.0 of the dslpermission library

implementation 'com.sagar: dslpermission: 2.0.0'

does not have the attribute
val requestCode: Int
that if it is in 2.0.1 of com.sagar: coroutinespermission and
com.sagar: livedatapermission,

the error is fixed when adding any of these, but technically it is not correct, please update the PermissionResul class for dslpermission ...

version 2.0.1
![image](https://user-images.githubusercontent.com/7816977/69916809-35b7fe80-143e-11ea-867c-cfc3a782faea.png)

version 2.0.0
![image](https://user-images.githubusercontent.com/7816977/69916871-0229a400-143f-11ea-888d-b77c1bda0ea6.png)

**Comment by LucianoFromTrelew** [comment]
There is no `com.sagar:dslpermission:2.0.1` :/

**Comment by sagar-viradiya** [comment]
@LucianoFromTrelew @AlbertoChamorro Thanks for reporting. Will release updated version of DSL soon.

**Comment by sagar-viradiya** [comment]
@AlbertoChamorro  @LucianoFromTrelew @adrynov 
I have pushed a fix for this Please review #34

**Comment by sagar-viradiya** [comment]
@AlbertoChamorro @LucianoFromTrelew @adrynov This should have been fixed in the latest release. Please check.
